nucypher/cli/actions.py

```diff
@@ -606,3 +607,24 @@ def get_or_update_configuration(emitter, config_class, filepath: str, config_opt
             emitter.message(f"Updated configuration values: {', '.join(updates)}", color='yellow')
             config.update(**updates)
         return emitter.echo(config.serialize())
+
+
+def select_worker_config_file(emitter, config_file, worker_address, provider_uri, network, federated):
+
+    config_root = abspath(dirname(config_file)) if config_file else DEFAULT_CONFIG_ROOT
+    any_worker_config = glob.glob(UrsulaConfiguration.default_filepath(config_root=config_root))
+
+    if not any_worker_config:
+        emitter.message("No Ursula configurations found.  run 'nucypher ursula init' then try again.", color='red')
+        raise click.Abort()
+
+    more_than_one_worker_config = glob.glob(f'{config_root}/ursula-0x*.json')
+    if more_than_one_worker_config:
+        if not config_file:
+            if not worker_address and not federated:  # TODO: Support Federated by walking the filesystem
```

Comment:
A little bit difficult to read many nested if's. Maybe change it to `if not(condition): return`?